bpf/lib/common.h

```diff
@@ -610,7 +610,9 @@ enum {
 #define DROP_INVALID_CLUSTER_ID	-192
 #define DROP_DSR_ENCAP_UNSUPP_PROTO	-193
 #define DROP_NO_EGRESS_GATEWAY	-194
+#define DROP_UNENCRYPTED_TRAFFIC -195
```

Comment:
I believe this one was added by mistake. Probably wouldn't have happened if I didn't decide to fix an indentation in the upstream commit :facepalm:

Reply:
ah good catch, thanks :man_facepalming: